[2024-07-30T10:27:51.682+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:27:51.722+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-21T00:00:00+00:00 [queued]>
[2024-07-30T10:27:51.735+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-21T00:00:00+00:00 [queued]>
[2024-07-30T10:27:51.736+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:27:51.845+0000] {taskinstance.py:2330} INFO - Executing <Task(LoadDimensionOperator): load_dimension_tables.load_users_dim_table> on 2018-11-21 00:00:00+00:00
[2024-07-30T10:27:51.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1262) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:27:51.854+0000] {standard_task_runner.py:63} INFO - Started process 1266 to run task
[2024-07-30T10:27:51.853+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'load_dimension_tables.load_users_dim_table', 'scheduled__2018-11-21T00:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmptsqc80sv']
[2024-07-30T10:27:51.855+0000] {standard_task_runner.py:91} INFO - Job 229: Subtask load_dimension_tables.load_users_dim_table
[2024-07-30T10:27:51.971+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-21T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:52.242+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='load_dimension_tables.load_users_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2018-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-21T00:00:00+00:00'
[2024-07-30T10:27:52.244+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:52.285+0000] {load_dim_operator.py:26} INFO - Loading dimension table users
[2024-07-30T10:27:52.317+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:52.354+0000] {sql.py:487} INFO - Running statement: 
                BEGIN;
                TRUNCATE TABLE users;
                
        INSERT INTO users (user_id, first_name, last_name, gender, level)
        SELECT DISTINCT ON (userid) userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
        ON CONFLICT (user_id) DO UPDATE
        SET first_name = EXCLUDED.first_name,
            last_name = EXCLUDED.last_name,
            gender = EXCLUDED.gender,
            level = EXCLUDED.level
    ;
                COMMIT;
            , parameters: None
[2024-07-30T10:27:52.820+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:52.841+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=load_dimension_tables.load_users_dim_table, run_id=scheduled__2018-11-21T00:00:00+00:00, execution_date=20181121T000000, start_date=20240730T102751, end_date=20240730T102752
[2024-07-30T10:27:52.955+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:53.007+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:27:53.058+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
